src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -2,16 +2,44 @@
 
 // The value of component scoped variables is always defined by a global variable:
 .pf-l-toolbar {
+  --pf-l-toolbar--MarginTop: var(--pf-global--spacer--sm);
+
   // Toolbar Section
   --pf-l-toolbar__section--MarginTop: var(--pf-global--spacer--md);
 
   // Toolbar Group
+  --pf-l-toolbar__group--MarginTop: var(--pf-global--spacer--sm);
+  --pf-l-toolbar__group--MarginBottom: var(--pf-global--spacer--sm);
   --pf-l-toolbar__group--MarginRight: var(--pf-global--spacer--xl);
+  --pf-l-toolbar__group--MarginRight-Zero: var(0);
```

Comment:
`var(0)` is invalid. A css var has to start with `--`, so "0" is an invalid name to use in `var()`. I think you just meant to use `--pf-l-toolbar__group--MarginRight-Zero: 0;`. And there are naming problems with `--pf-l-toolbar__group--MarginRight-Zero` that I'll talk about more below.  You can reference how a variable is named in the [variables section of the pf4 guidelines](https://pf-next.com/guidelines#variables)